### PR TITLE
Dawn redesign

### PR DESCRIPTION
**Pull request by anguyn12**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,83 +4,219 @@
...
-					<img class="logo-img" src="./images/logo.png">
-					<h1 class="logo pop">
-						Virtision
-					</h1>
-					<h4 class="motto">Easily create, share and enjoy immersive worlds together</h4>
+					<h1 class="logo pop">The XR Social Music Platform</h1>
+					<h2 class="motto">Easily create, share and enjoy immersive worlds together</h2>
 					<div class="subscribe">
 						<h5 class="info-text">
 							Join the waiting list for the beta. We keep you posted.
```
Comment (parseccentric) [inline index.html]: We -> We'll

File: index.html [hunk 2]
```diff
@@ -4,83 +4,219 @@
...
+				</div>
+
+			</div>
+		</div>
+	</div>
+	<div class="main">
+		<div class="container value">
+			<div class="row justify-content-center">
+				<div class="col-lg-8 center">
+					<h1 class="pop"> For any creatives at any level to create and share their imagination in XR</h1>
```
Comment (parseccentric) [inline index.html]: Grammatically correct, and I like what this sentence is saying, but I maybe we could make it sound even better? 

"Creatives, from novice to expert, can share their imagination in XR."
"Calling all creatives of all skill levels: share your imagination in XR."
"For creatives to share their imagination in XR, whether they're beginners or experts."

Also maybe creatives -> creators? IDK.

File: index.html [hunk 3]
```diff
@@ -4,83 +4,219 @@
...
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row pt-5">
+				<div class="col-lg-6">
+					<img class="demo" src="./images/creation.png" width="400">
+				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Simply create your XR story with just a few taps</h1>
+					<h3 class="motto">experience a unified and simplified creation process to unblock your creativity</h3>
```
Comment (parseccentric) [inline index.html]: This is good. Just another wording tweak. 

"Our seamless creation process will unblock your creativity."
"Use our unified creation process to unblock your creativity."
"Our simplified creation process won't block your creativity."

File: index.html [hunk 4]
```diff
@@ -4,83 +4,219 @@
...
+		</div>
+		<div class="container value">
+			<div class="row">
+				<div class="col-lg-6 order-md-2 order-sm-1">
+					<img class="demo" src="./images/sanbox.png" width="400">
+					<!--<a href='https://www.freepik.com/psd/woman'>Woman psd created by freepik - www.freepik.com</a>-->
+				</div>
+				<div class="col-lg-6  order-md-1 order-sm-2">
+					<h1 class="pop">The XR sandbox for your creativity</h1>
+					<h3 class="motto">powered by the community-built 3D assets, music, animation and effects library</h3>
```
Comment (parseccentric) [inline index.html]: -> powered by a community-built library of 3D assets, music, animations, and effects

File: index.html [hunk 5]
```diff
@@ -4,83 +4,219 @@
...
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row">
 				<div class="col-lg-6">
-					<video playsinline="" muted="" class="img-rounded video" src="images/demo.mp4" style="width:inherit" loop autoplay=""></video>
+					<img class="demo" src="./images/alldevice2.png" width="400">
 				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Share and enjoy many XR creations together in any device</h1>
```
Comment (parseccentric) [inline index.html]: -> Share and enjoy many XR creations together **on** any device.

Kinda feel like this one could use a little overhaul.

"Share your XR creations with any device" 
"Share and enjoy a universe of XR creations with any device"
"Share and enjoy a world of XR creations with any device"

File: index.html [hunk 6]
```diff
@@ -4,83 +4,219 @@
...
+					<h3 class="motto">remove the barrier between your XR arts and your audience</h3>
```
Comment (parseccentric) [inline index.html]: -> Remove **barriers** between your XR **art** and your audience

Or "arts" -> "artworks"

File: index.html [hunk 7]
```diff
@@ -4,83 +4,219 @@
...
 
+				</div>
 			</div>
+		</div>
+		<div class="end rounded shadow-lg">
+			<div class="container value mt-5">
+				<div class="row justify-content-center">
+					<div class="col-lg-9 center-close mt-5">
+						<h1 class="pop">Bring more people to contribute to the future of XR</h1>
```
Comment (parseccentric) [inline index.html]: -> Bring more people **into contributing** to the future of XR

File: index.html [hunk 8]
```diff
@@ -4,83 +4,219 @@
...
+						<h3 class="motto">Our strength lies in the hands of our users, forming a mobile XR community welcoming everyone of all levels and backgrounds to share their creative story through art and music.
```
Comment (parseccentric) [inline index.html]: Overall good, but kinda aimed towards VCs. Maybe something more creator oriented? 

We envision building a mobile XR community with our users, welcoming everyone of all levels and backgrounds to share their creative story through art and music.

File: index.html [hunk 9]
```diff
@@ -4,83 +4,219 @@
...
 		</div>
-		<div class="footer">
-			<div class="container">
-				© 2021 Virtision. All Rights Reserved.
 
-			</div><!-- /.navbar-collapse -->
+		<div class="container value mt-5">
+			<div class="row justify-content-center">
+				<div class="col-lg-12 center">
+					<h1 class="pop"> Be the first to get the app </h1>
```
Comment (parseccentric) [inline index.html]: "Be one of the first to get the app"
"Get the app early"
"Be first to get the app"

File: index.html [hunk 10]
```diff
@@ -4,83 +4,219 @@
...
+			</div>
 		</div>
+
+
+
 	</div>
+
+
+	<div class="footer">
+		<div class="container">
```
Comment (parseccentric) [inline index.html:213]: it'd be nice if this had a background so it wouldn't just transparently go right over the body text
 
 but it's a super minor thing and not really that important right now

**Review by parseccentric — CHANGES_REQUESTED** [review]
comments were left as single comments

File: index.html [hunk 11]
```diff
@@ -4,83 +4,219 @@
...
-								<button type="submit" class="btn btn-email btn-fill">Notify Me</button>
-
 							</form>
-							<div class="media">
-								<a href="#">
+
+							<div class="media mb-3">
+								<a href="https://www.linkedin.com/company/virtision-com">
 									<i class="fa fa-linkedin-square"></i>
 								</a>
```
Comment (parseccentric) [inline index.html:96]: Screen Shot 2021-04-28 at 11 56 54 PM

Why's there these underscores after the LI and FB icons?

Comment (anguyn12) [reply index.html:96]: yeah for some reason it's show like that, I will change the email icon to instagram icon instead

File: index.html (hunk 2 above)
Comment (anguyn12) [reply index.html]: yeah creators is better since it is one of the popular keywords for social media nowadays

File: index.html (hunk 3 above)
Comment (anguyn12) [reply index.html]: I decide to change to this: "our seamless creation experience will unblock your creativity."

File: index.html (hunk 4 above)
Comment (anguyn12) [reply index.html]: updated

File: index.html (hunk 5 above)
Comment (anguyn12) [reply index.html]: I choose this ""Share and enjoy a universe of XR creations with any device"

File: index.html (hunk 6 above)
Comment (anguyn12) [reply index.html]: I turn to this "remove barriers between your XR artworks and your audience"

File: index.html (hunk 7 above)
Comment (anguyn12) [reply index.html]: done

File: index.html (hunk 8 above)
Comment (anguyn12) [reply index.html]: yub true. updated

File: index.html (hunk 9 above)
Comment (anguyn12) [reply index.html]: "Be one of the first to get the app"

File: index.html (hunk 11 above)
Comment (anguyn12) [reply index.html:96]: weird the instagram isnt like that either

Comment (anguyn12) [reply index.html:96]: oh text decoration fixed

File: index.html (hunk 1 above)
Comment (anguyn12) [reply index.html]: ok

File: index.html [hunk 12]
```diff
@@ -4,83 +4,219 @@
...
+
+
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
 	<link href="css/main.css" rel="stylesheet" />
 
 	<!--     Fonts     -->
-	<link href="https://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet">
-	<script src="js/jquery-1.10.2.js" type="text/javascript"></script>
-	<script src="js/bootstrap.min.js" type="text/javascript"></script>
+	<link href="http://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet">
```
Comment (Virtision) [inline index.html]: need to use HTTPS for this one. original was https, so just use the old link

**Review by Virtision — CHANGES_REQUESTED** [review]

File: index.html (hunk 12 above)
Comment (anguyn12) [reply index.html]: okay

**Pull request merged**